test/Microsoft.DotNet.Docker.Tests/VerifyImageDescriptor.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.Docker.Tests
+{
+    public class VerifyImageDescriptor
```

Comment:
Minor - `ImageDescriptior` seems to be a better name for this class.

Reply:
Good suggestion - changed